src/Sulu/Bundle/SecurityBundle/Tests/Unit/User/UserProviderTest.php

```diff
@@ -108,4 +115,10 @@ public function testRefreshUserWithDisabledUser(): void
         $this->user->setEnabled(false);
         $this->userProvider->refreshUser($this->user);
     }
+
+    public function testUpgradePassword(): void
+    {
+        $this->userProvider->upgradePassword($this->user, 'hashedPass');
+        $this->assertEquals('hashedPass', $this->user->getPassword());
```

Comment:
```suggestion
        $this->assertSame('hashedPass', $this->user->getPassword());
```

Prefer assertSame over assertEquals